docs/tutorials/tabular_prediction/tabular-quickstart.md

```diff
@@ -76,20 +79,67 @@ AutoGluon automates this process.
+print(f"Predicted probabilities of class '{positive_class}':", pred_probs)
+```
+
+Besides inference, this object can also summarize what happened during fit.
 
 ```{.python .input}
 results = predictor.fit_summary()
 ```
 
-From this summary, we can see that AutoGluon trained many different types of models as well as an ensemble of the best-performing models.  The summary also describes the actual models that were trained during fit and how well each model performed on the held-out validation data.  We can also view what properties AutoGluon automatically inferred about our prediction task:
+From this summary, we can see that AutoGluon trained many different types of models as well as an ensemble of the best-performing models.  The summary also describes the actual models that were trained during fit and how well each model performed on the held-out validation data.  We can view what properties AutoGluon automatically inferred about our prediction task:
 
 ```{.python .input}
 print("AutoGluon infers problem type is: ", predictor.problem_type)
-print("AutoGluon categorized the features as: ", predictor.feature_metadata.type_map_raw)
+print("AutoGluon identified the following types of features:")
+print(predictor.feature_metadata)
+```
+
+AutoGluon correctly recognized our prediction problem to be a **binary classification** task and decided that variables such as `age` should be represented as integers, whereas variables such as `workclass` should be represented as categorical objects. The `feature_metadata` attribute allows you to see the inferred data type of each predictive variable after preprocessing (this is it's *raw* dtype; some features may also be associated with additional *special* dtypes if produced via feature-engineering, e.g. numerical representations of a datetime/text column).
+
+We can evaluate the performance of each individual trained model on our (labeled) test data:
+```{.python .input}
+predictor.leaderboard(test_data, silent=True)
+```
+
+When we call `predict()`, AutoGluon automatically predicts with the model that displayed the best performance on validation data (i.e. the weighted-ensemble). We can instead specify which model to use for predictions like this:
+```
+predictor.predict(test_data, model='NeuralNetClassifier')
+```
+
+Above the scores of predictive performance were based on a default evaluation metric (accuracy for binary classification). Performance in certain applications may be measured by different metrics than the ones AutoGluon optimizes for by default. If you know the metric that counts in your application, you should specify it as demonstrated in the next section.
+
+## Maximizing predictive performance
+
+To get the best predictive accuracy with AutoGluon, you should generally use it like this:
+
+```{.python .input}
+time_limits = 60 # for quick demonstration only, you should set this to longest time you are willing to wait (in seconds)
+metric = "roc_auc" # specify your metric here
```

Comment:
nit: for consistency we should maybe exclusively use `'` instead of occasionally using `"`.